qa/suites/upgrade/luminous-x/ceph-deploy/luminous-mimic.yaml

```diff
@@ -0,0 +1,77 @@
+meta:
+- desc: |
+    Setup 4 node ceph cluster using ceph-deploy, use latest
+    stable luminous as initial release, upgrade to mimic (ATM to master) and
+    also setup mgr nodes along after upgrade, check for
+    cluster to reach healthy state, After upgrade run kernel tar/untar
+    task and systemd task. This test will detect any
+    ceph upgrade issue and systemd issues.
```

Comment:
should be 'ceph upgrade issues'.

Reply:
@amitkumar50 fixed on master but will not backport :)